### ISSUE TITLE
TypeError: Cannot read property 'add' of undefined

### ISSUE THREAD

**Issue by oojacoboo** [bmewburn/intelephense#179]
Intelephense continually crashes after restarting itself 3x and hitting the following `TypeError`.  It's unusable.

```
[Info  - 5:20:07 PM] Initialising
[Info  - 5:20:07 PM] Initialised in 8 ms
[Info  - 5:20:07 PM] Intelephense 1.0.1
[Info  - 5:20:07 PM] Indexing started.
[Warn  - 5:20:08 PM] file:///Volumes/repos/xxxx/api/app/lib/model/BaseRole.php is over the maximum file size of 1000000 bytes.
[Info  - 5:21:44 PM] Indexing ended. Elapsed 96s.
[Info  - 5:21:44 PM] Writing state to /Users/jacob/Library/Application Support/Code/User/workspaceStorage/e92a4f6f6e31f38c1b4439363a91e35b/bmewburn.vscode-intelephense-client/21b28655.
[Info  - 5:21:52 PM] Finished writing state to /Users/jacob/Library/Application Support/Code/User/workspaceStorage/e92a4f6f6e31f38c1b4439363a91e35b/bmewburn.vscode-intelephense-client/21b28655 in 8.1s
/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39
 */!function(e){var t={};function n(i){if(t[i])return t[i].exports;var r=t[i]={i:i,l:!1,exports:{}};return e[i].call(r.exports,r,r.exports,n),r.l=!0,r.exports}n.m=e,n.c=t,n.d=function(e,t,i){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:i})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var i=Object.create(null);if(n.r(i),Object.defineProperty(i,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var r in e)n.d(i,r,function(t){return e[t]}.bind(null,r));return i},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.

TypeError: Cannot read property 'add' of undefined
    at f.preorder (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:339821)
    at t.MasterDiagnosticsVisitor.preorder (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:334567)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3568)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.visitNode (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:3653)
    at p.traverse (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:1604)
    at diagnosticsTransform.e (/Users/jacob/.vscode/extensions/bmewburn.vscode-intelephense-client-1.0.1/node_modules/intelephense/lib/intelephense.js:39:259587)
```

**Comment by M-Porter** [comment]
Same crashing is occurring on hover:
```
[Error - 4:03:15 PM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'flush' of undefined
  Code: -32603 
```

Seems to be related to the `1.0.0` and `1.0.1` release that was pushed out an hour ago.

Downgrading to `0.8.8` fixed these issues for me.

**Comment by bdenizar** [comment]
Hello,

Same here:
`TypeError: Cannot read property 'add' of undefined
    at f.preorder (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:339821)
    at t.MasterDiagnosticsVisitor.preorder (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:334567)
    at p.visitNode (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:3568)
    at p.visitNode (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:3653)
    at p.visitNode (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:3653)
    at p.traverse (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:1604)
    at diagnosticsTransform.e (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:259587)
    at t.Debounce.y.create [as handler] (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:260185)
    at Timeout._timer.setTimeout [as _onTimeout] (c:\Users\ben\.vscode\extensions\bmewburn.vscode-intelephense-client-1.0.1\node_modules\intelephense\lib\intelephense.js:39:4457)
    at ontimeout (timers.js:427:11)
[Info  - 10:24:59] Connection to server got closed. Server will restart.`

**Comment by bmewburn** [comment]
fixed in 1.0.2
